### PR TITLE
Initial commit of Personas

### LINKED ISSUE #3: Create Initial User Persona Document
- [x] Develop 2-3 user personas based on the target audience for the educational tool, including their backgrounds, goals, and potential use cases.
- [ ] Additionally, include a "User Personas" tab in the header of the home page where these personas can be displayed. This will help stakeholders and users understand who the tool is designed for and the specific needs being addressed.

**Why it's useful:** Helps the team understand and focus on the needs of the end-users throughout development.
**Effort:** Can be completed in about 2 hours with some research and creativity.

### PR DESCRIPTION
**Pull request by tluke900**
Fixes #3 
Added 3 personas and their main use cases and goals to a markdown File. 
Added User Personas Button to the HTML code.
Both of these changes were added to fulfill Issue 3.
One new button object added to HTML, links to separate text file containing Persona documentation.

### CODE REVIEW

File: Personas Doc.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## User Persona Document
+
+### Persona 1:
```
Comment (Balijah) [inline Personas Doc.md:3]: Personas could possibly include more detail as to who would need this program, why they would need this program, how they could use it in their field, where they could use it (schools, institutions, etc.), and when they would be able to use it (availability).

File: Personas Doc.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Goals: Her ultimate goal would be to use the program to help facilitate the learning of her students, as well as bridge a gap that current exists of needing models or actual bones from donors.
+
+Primary Use: Assigning work that would best be completed through use of Bone Box. The professor would recommend use of the program so that
+
+### Persona 2:
+
+Background: A student studying medicine with the hope to become a general practitioner after graduation.
+
+Goals: His primary goal would be to ustalize the program to gain further understanding about each of the bones.
```
Comment (Balijah) [inline Personas Doc.md:15]: **ustalize

**Comment by kate-holdener** [comment]
> Pull Request to satisfy the first task of Issue 3. Added 3 personas and their main use cases and goals.

Please use:
Fixes #3
At the top of the PR.

**Review by DeviPriya-Kolla — COMMENTED** [review]
**Preliminary Step:**

- Consult with the tech lead for a detailed orientation on the issue
- Clarify any uncertainties about requirements or implementation details

**Code Review Summary:**

- User personas created but not integrated into the app.
- Missing "User Personas" tab in header.
- Standalone markdown file instead of proper code integration.
- PR deviates from skeleton code.
- Incomplete PR description (missing issue link, change details).

**To-Do:**

- Integrate personas into existing code structure.
- Add "User Personas" tab to header.
- Display personas in the app.
- Update PR description with full details.
- Follow project structure and address all issue requirements.

**Pull request closed without merging**